01-Login/package.json

```diff
@@ -6,7 +6,7 @@
     "dotenv": "^6.1.0",
     "ejs": "^2.7.4",
     "express": "^4.17.1",
-    "express-openid-connect": "^0.6.0",
+    "express-openid-connect": "^1.0.1",
```

Comment:
Asking since the repo lacks tests. Was there any breaking change involved? Have you run it locally?

Reply:
I did yes ... thanks for asking, though!